sherlock_homes.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
 
   # Runtime
   spec.add_dependency 'geocoder', '~> 1.2'
+  spec.add_dependency 'activesupport', '~> 4.2'
```

Comment:
Is this high a level of activesupport required?  Is it just being used for autoload?  Let's try to lower this requirement as far as possible for older Rails users.